src/js/pages/SundarGutka/index.js

```diff
@@ -32,6 +32,9 @@ class SundarGutka extends React.PureComponent {
   };
 
   render() {
+    requestAnimationFrame(() => {
+      window.scrollTo(0, 0);
+    });
```

Comment:
This is bad. render shouldn't have sideeffects. 

Do it in componentDidUpdate instead. Further reading;

https://reacttraining.com/react-router/web/guides/scroll-restoration

Reply:
Fixed. @bogas04